tests/all_tests_v2.nim

```diff
@@ -19,6 +19,10 @@ import
   ./v2/waku_archive/test_retention_policy,
   ./v2/waku_archive/test_waku_archive
 
+# TODO: add the postgres test when we can mock the database.
```

Comment:
In case you wanted to have tests enabled in CI, you could just exec a `docker run` command from tests to setup DB prior execution the tests themselves - it should work fine in CI

Reply:
Hey @vpavlin ! In the end I will tackle that in a separate PR because I need to tweak it in a way that the postgres tests are only run in a Linux platform. This is because the GH Actions only allow starting a docker container within Linux runners.
Thanks for the hint !